### ISSUE TITLE
verify_as_json use jsondiff

### ISSUE THREAD

**Issue by valentinedwv** [approvals/ApprovalTests.Python#146]
Rather than just a text compare, would an option to use the jsondiff or deepdif modules on json files.

I have a sparql query where the results come back if differing orders.... humm, guess I need to add some form of ordering in the query.

**Comment by isidore** [comment]
yes, that would be good, if you join on sundays we could take a look together to make it sort out of the box

**Comment by JayBazuzi** [comment]
Closing as json is now sorted
